cishouseholds/edit.py

```diff
@@ -792,3 +793,29 @@ def edit_to_sum_or_max_value(
         .cast("integer"),
     )
     return df
+
+
+def add_work_from_home_identifier(
+    df: DataFrame,
+    columns_to_check_in: List[str],
+    regex_pattern: str = "(W(K|ORK.*?) F(ROM?) H(OME?))|(WFH)",
+    pattern_result_column: str = "is_wfh",
```

Comment:
We tend to avoid using acronyms, so would prefer "working_from_home" here please

As a project standard, we also tend to name this parameter "column_name_to_assign" for consistency between functions. Can see examples in derive.py